src/controllers/User-Sign-Up-Controller.coffee

```diff
@@ -93,5 +120,10 @@ class User_Sign_Up_Controller
   render_Page: (jade_Page,params)=>
     @.res.send @.jade_Service.render_Jade_File jade_Page, params
 
-
+  loadSecretFile:() ->
+    if (process.cwd().path_Combine('../../config/SiteData_TM/secrets.json').file_Exists())
+      secrets = process.cwd().path_Combine('../../config/SiteData_TM/secrets.json').load_Json()
+      return secrets
```

Comment:
Actually that code is not longer needed since the secrets are read from the config file.